docker/registry/internal/acr.go

```diff
@@ -29,16 +29,16 @@ func (c *azureContainerRegistry) Match() bool {
 	return strings.Contains(c.repoDetails.ServerAddress, "azurecr.io")
 }
 
-func getUserNameFromAuth(auth string) (string, error) {
+func unpackAuthToken(auth string) (string, string, error) {
```

Comment:
Let's add a comment or use named return params to explain what (string, string) is